airbyte-server/src/test/java/io/airbyte/server/handlers/ConnectionsHandlerTest.java

```diff
@@ -82,6 +88,14 @@
   private StandardSyncOperation standardSyncOperation;
   private WorkspaceHelper workspaceHelper;
   private TrackingClient trackingClient;
+  private TemporalWorkerRunFactory temporalWorkflowHandler;
+  private SyncJobFactory jobFactory;
+  private JobPersistence jobPersistence;
+  private LogConfigs logConfigs;
+  private FeatureFlags featureFlags;
+
+  // TODO: bmoric move to a mock
```

Comment:
what's stopping you today?

Reply:
A refactoring of the test. Currently the `ConnectionsHandlerTest` is testing both the `ConnectionHandler` and the `ConnectionHelper` which contains methods that have been moved from the `ConnectionHandler`. Using a mock today will actually reduce the coverage.